### PR TITLE
vollkorn: init at 4.105

### PR DESCRIPTION
**Pull request by rasendubi**
###### Motivation for this change
I want to use this font.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/vollkorn/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    unzip -j $downloadedFile \*.otf   -d $out/share/fonts/opentype
+    unzip -j $downloadedFile \*.woff  -d $out/share/fonts/woff
+    unzip -j $downloadedFile \*.woff2 -d $out/share/fonts/woff2
+    unzip -j $downloadedFile \*.eot   -d $out/share/fonts/EOT
+    unzip -j $downloadedFile \*.txt   -d "$out/share/doc/${name}"
+  '';
+
+  sha256 = "0pvy72v8wsnghww67vy8i1wyk4nikyh8pylk7gjp21ar754sih3k";
+
+  meta = with lib; {
```
Comment (puzzlewolf) [inline pkgs/data/fonts/vollkorn/default.nix]: A package needs a `maintainer` :)

Please add yourself (as you created the package), and also add yourself to maintainers/maintainer-list.nix in a separate commit.

**Review by puzzlewolf — CHANGES_REQUESTED** [review]
##### Reviewed points

- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] package build on x86_64-linux
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.platforms` is set
- [ ] `meta.maintainers` is set
- [x] source is fetched using the appropriate function
- [x] phases are respected

**Comment by rasendubi** [comment]
@puzzlewolf updated, thanks!

**Comment by rasendubi** [comment]
@GrahamcOfBorg eval

**Review by puzzlewolf — APPROVED** [review]
LGTM :+1:

**Review by puzzlewolf — CHANGES_REQUESTED** [review]
Oh, I just noticed something:

```
$ ls share/doc/vollkorn-4.105/ -a
.  ..  ._Fontlog.txt  Fontlog.txt  ._OFL-FAQ.txt  OFL-FAQ.txt  OFL.txt
```
I don't think we want `._Fontlog.txt` and `._OFL-FAQ.txt`:
```
$ file -s share/doc/vollkorn-4.105/._*
share/doc/vollkorn-4.105/._Fontlog.txt: AppleDouble encoded Macintosh file
share/doc/vollkorn-4.105/._OFL-FAQ.txt: AppleDouble encoded Macintosh file
```
Bonus points for reporting upstream, I doubt they should be in the release .zip :)

**Comment by rasendubi** [comment]
We need to re-package vollkorn to use mkDerivation instead of fetchzip directly (as pointed in #83537).

I stopped using this font so I have little motivation to do that and I'm closing this PR.

If anyone wants vollkorn packaged for nixpkgs, just ping me here and I'll do that.

**Pull request closed without merging**